compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6902,6 +6902,20 @@ bool OMR::Power::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&
          isGetThenUpdate = true;
          doInline = true;
          }
+      else if (comp->getSymRefTab()->isNonHelper(symRef, TR::SymbolReferenceTable::atomicFetchAndAdd8BitSymbol))
+         {
+         isAddOp = true;
+         isLong = false;
```

Comment:
The implementation of [`inlineSimpleAtomicUpdate`](https://github.com/eclipse/omr/blob/daa7c85bae82a3f72af5053b1dc51f5f6217f20d/compiler/p/codegen/OMRTreeEvaluator.cpp#L6700) will need to be revised to handle 8 and 16 operands for atomic fetch-and-add for Power.

Reply:
My mistake, it was not my intention to turn these on for power! I'll remove these